Lib/importlib/_bootstrap.py

```diff
@@ -26,6 +26,13 @@ def _object_name(obj):
     except AttributeError:
         return type(obj).__qualname__
 
+def _normalize(name):
+    """ Normalize 'name' in NKFC form """
+    global _unicodedata_normalize
+    if _unicodedata_normalize is None:
+        from unicodedata import _normalize
```

Comment:
```suggestion
        from unicodedata import _normalize as _unicodedata_normalize
```

Reply:
I get an import error for `_normalize` but not `normalize` in the REPL. Is that expected?